src/Core/src/Core/MauiApp.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Maui
+{
+	public sealed class MauiApp : IHost
```

Comment:
MaulApplication?

```suggestion
	public sealed class MauiApplication : IHost
```

Reply:
Unfortunately there are already many various types in MAUI called MauiApplication or similar, which contain the user's application entry point on each platform. I don't like the name `MauiApp` so for now it's a placeholder until we can figure out the right names for everything.

Reply:
I see.

Reply:
Since it is IHost, we can use that word. It is not the app, so MauiAppHost or MauiHost?